switchboard/scripts/sqlx-precommit-setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env sh
```

If only to avoid diverging from how the other scripts work.